static/docs.css

```diff
@@ -102,14 +102,26 @@ a.sc-logo img {
 }
 
 .side-nav .nav-header {
-  color: #e6522c;
-  text-transform: uppercase;
-  font-size: 16px;
   display: block;
   margin: 20px auto 15px auto;
+  font-size: 16px;
+}
+
+.side-nav .nav-header a {
+  color: #e6522c;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.side-nav ul {
+	display: none;
```

Comment:
You're using tabs for indentation here, while the rest of the file uses two spaces. Same below.